hs_err_pid23425.log

```diff
@@ -0,0 +1,86 @@
+#
```

Comment:
included by accident, I assume?

Reply:
Yep, `topic/multithreading` is a staging area for our experiments on multithreading, we're going to remove this before merging to master.